setup.py

```diff
@@ -30,11 +30,11 @@
     packages=find_packages(exclude=('tests*',)),
     namespace_packages=('intuitlib',),
     install_requires=[
-        'python_jose>=2.0.2',
         'requests>=2.13.0',
         'requests_oauthlib>=1.0.0',
         'six>=1.10.0',
         'enum-compat',
+        'oic==1.7.0',
```

Comment:
```suggestion
        'oic',
```

Reply:
Yes, please don’t pin the version, libraries should be liberal to allow upgrades.